sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -127,6 +127,16 @@ def wrapper(self, *args, **kwargs):
   return frame_base.with_docs_from(base)(wrapper)
 
 
+# Docstring to use for head and tail (commonly used to peek at datasets)
+_PEEK_METHOD_EXPLANATION = (
+    "because it is `order-sensitive <https://s.apache.org/dataframe-order-sensitive-operations>`_.\n\n"
+    "If you'd like to use it to peek at a large dataset, interactive Beam's "
```

Comment:
I prettied this up a little bit, since we're using it in a bunch of places. But just a suggestion. No need to change:

"If you want to peek at a large dataset, consider using  :func:`ib.collect <apache_beam.runners.interactive.interactive_beam.collect>` with ``n`` specified. If you want to find the N largest elements, consider using :meth:`DeferredDataFrame.nlargest`."

Reply:
Done, thank you!